tarodb/asset_minting_test.go

```diff
@@ -76,6 +74,77 @@ func assertSeedlingBatchLen(t *testing.T, batches []*tarogarden.MintingBatch,
 	require.Len(t, batches[0].Seedlings, numSeedlings)
 }
 
+// storeGroupGenesis generates a group genesis asset and inserts it into the DB.
+// The group genesis asset information needs to be in the DB before reissuance.
+func storeGroupGenesis(t *testing.T, ctx context.Context,
```

Comment:
Hmm, I would've at least expected some standalone unit tests for the new queries (`FetchGenesisID`, `FetchGroupByGenesis` and `FetchGroupByGroupKey`). Those should be pretty straightforward to add I believe? As not much interaction with the seedlings themselves are required.